tests/tracking/_default_experiment/test_databricks_job_context.py

```diff
@@ -0,0 +1,59 @@
+from unittest import mock
+
+from mlflow.entities import SourceType
+from mlflow.utils.mlflow_tags import (
+    MLFLOW_DATABRICKS_JOB_TYPE_INFO,
+    MLFLOW_EXPERIMENT_SOURCE_TYPE,
+    MLFLOW_EXPERIMENT_SOURCE_ID,
+)
+from mlflow.tracking._default_experiment.databricks_job_context import (
+    DatabricksJobExperimentContext,
+)
+from tests.helper_functions import multi_context
+from mlflow.tracking import MlflowClient
+
+
+def test_databricks_job_default_experiment_in_context():
+    with mock.patch(
+        "mlflow.utils.databricks_utils.is_in_databricks_job"
+    ) as in_job_mock, mock.patch(
+        "mlflow.utils.databricks_utils.get_job_type_info"
+    ) as get_job_type_info:
+        in_job_mock.return_value = True
+        get_job_type_info.return_value = "NORMAL"
+        assert DatabricksJobExperimentContext().in_context() == True
```

Comment:
Can we add a negative test case asserting that `in_context()` returns `False` by default and when `is_in_databricks_job` returns `False` and when `get_job_type_info` returns something other than `NORMAL`?